imgee/views/files.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+from flask import (flash, g, jsonify, redirect, render_template, request, url_for)
+from sqlalchemy import and_
+
+from coaster.views import load_model, load_models
+from imgee import app, forms, lastuser
```

Comment:
Maybe import just the relevant classes and functions from `forms`, instead of importing everything?